docs_user/modules/proc_using-new-subnet-ranges.adoc

```diff
@@ -46,59 +26,53 @@ network_config:
       next_hop: 192.168.1.1
     - ip_netmask: 172.31.0.0/24 <1>
       next_hop: 192.168.1.100 <2>
-```
+----
 <1> The new control plane subnet.
 <2> The control plane IP address of the adopted data plane node.
-
-Do the same for other networks that will need to use different subnets for the
-new and old parts of the deployment.
-
-Once done, run `openstack overcloud node provision` to apply the new configuration.
-
-Note that network configuration changes are not applied by default to avoid
-risk of network disruption. You will have to enforce the changes by setting the
++
+Repeat this step for other networks that need to use different subnets for the new and old parts of the deployment.
+
+. Apply the new configuration to every adopted node:
++
+----
+$ openstack overcloud node provision
+--stack <stack>
+--network_config
+----
++
+[NOTE]
+Network configuration changes are not applied by default to avoid
+the risk of network disruption. You must enforce the changes by setting the
 `StandaloneNetworkConfigUpdate: true` in the {OpenStackPreviousInstaller} configuration files.
 
-Once `openstack overcloud node provision` is complete, you should see new link local routes to the
-new subnet on each node. For example,
-
-```bash
+. Confirm that there are new link local routes to the new subnet on each node. For example:
++
+----
 # ip route | grep 172
 172.31.0.0/24 via 192.168.122.100 dev br-ctlplane
-```
-
----
-
-The next step is to configure similar routes for the old subnet for control plane services attached to the networks. This is done by adding `routes` entries to
-`NodeNetworkConfigurationPolicy` CRs for each network. For example,
+----
 
-```yaml
+. To configure similar routes for the old subnet for the control plane services that are attached to the networks, add `routes` entries to the `NodeNetworkConfigurationPolicy` CRs for each network. For example:
++
+----
       - destination: 192.168.122.0/24 <1>
         next-hop-interface: ospbr <2>
-```
-<1> The isolated network's original subnet on the data plane.
-<2> The {OpenShiftShort} worker network interface that corresponds to the isolated network on the data plane.
-
-Once applied, you should eventually see the following route added to your {rhocp_long} nodes.
-
-```bash
+----
+<1> The original subnet of the isolated network on the data plane.
+<2> The {rhocp_long} worker network interface that corresponds to the isolated network on the data plane.
++
+As a result, the following route is added to your {OpenShiftShort} nodes:
++
+----
 # ip route | grep 192
```

Comment:
I now think we may want to avoid advising customers to run commands directly on CoreOS shell, as I think it's generally discouraged in OpenShift world. Perhaps the `ip route` and `ping` checks here should be avoided. I wonder what @jistr would say.

Reply:
@jistr ^